### PR TITLE
Weekly security link dump for the week of May 25th

### PR DESCRIPTION
**Pull request by cneill**
__Bio:__
Charles Neill is a Security Engineer at Rackspace, who tests Rackspace products as well as OpenStack projects like Barbican. He is also a developer on [DefectDojo](https://github.com/rackerlabs/django-DefectDojo), an open source defect tracking system using the Django framework.

__Social Media:__
Twitter: [@ccneill](https://twitter.com/ccneill)
Github: [cneill](https://github.com/cneill)
Website: [inventropy.us/blog/](https://inventropy.us/blog/)

__Editing Preference:__
Feel free to edit as needed.

__Posting Preference:__
Desired date to publish: May 27th, 2015

If you guys are tired of me posting these (I notice that they will constitute the 3 most recent posts on the developer blog at the time of this PR), just let me know. If they're not garnering an audience, I'm fine with just posting them internally and finding other things to talk about publicly.

Thanks!

### CODE REVIEW

File: src/site_source/_posts/2015-05-25-weekly-security-link-dump-may-25th.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+# Vulnerabilities
+
+- [__LogJam__][logjam] - The big news last week was a vulnerability exploiting "export ciphers," somewhat similar to the recent FREAK vulnerability. This particular issue affects the Diffie-Hellman key exchange, and could be exploited by an attacker with a privileged network position. The idea is to modify the cipher list sent by the client to only include export-grade cipher suites, which then forces D-H key exchange to happen with a weaker 512-bit modulus, which can be broken by the attacker, especially if it is using the default D-H parameters provided by the server's SSL library. [This answer on Security Stack Exchange][logjam2] provides some more detail about the vulnerability itself, and what to do about it. [Cloudflare][logjam3] also has a great post describing the D-H key exchange process, and how this vulnerability breaks it.
```
Comment (catlook) [inline src/site_source/_posts/2015-05-25-weekly-security-link-dump-may-25th.md]: ... 512-bit modulus. This weaker modulus can be ...
... vulnerability itself and what to do ...
... exchange process and how this vulnerability ...

File: src/site_source/_posts/2015-05-25-weekly-security-link-dump-may-25th.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+- [__IPsec vulnerabilities and software security prediction__][ipsec] - A researcher who goes by the handle "Javantea" has posted a long article that briefly describes a denial-of-service attack on the IKE daemon (part of IPsec), and he suggests that IPsec is now essentially broken. The author's bigger point is not about IPsec itself, but about poorly-maintained projects that nonetheless end up on a large number of machines. He dives into a process for filtering out insecure packages from Linux distros, such as measures of active maintenance and number of known security issues, and lists several packages that he believes should be removed from distros' package repositories unless they get better at proactively fixing security issues.
```
Comment (catlook) [inline src/site_source/_posts/2015-05-25-weekly-security-link-dump-may-25th.md]: ... IPsec itself but about ...

File: src/site_source/_posts/2015-05-25-weekly-security-link-dump-may-25th.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+
+# Tools
+
+- [__Regexr__][regexr] - A nifty online tool for creating and evaluating regular expressions, with lots of helpful documentation and the ability to run replacements.
+
+# Random Link of the Week
+
+- [__PowerPoint should be banned__][ppt] - This opinion piece from Katrin Park at the Washington Post argues that PowerPoint should be banned from use by enterprises and other organizations, due to the limiting of conversation and thoughtful examination that PowerPoint enforces. She calls out the Columbia shuttle disaster as an example of vital information being hidden in tiny bullet points, which may have played a part in why NASA took the unfortunate course of action that they did. Whether you agree or disagree with the entire premise, I'd bet a world with fewer PowerPoint decks and more in-depth conversations is probably a better one on balance.
+
```
Comment (catlook) [inline src/site_source/_posts/2015-05-25-weekly-security-link-dump-may-25th.md]: [Edits complete :)]

**Comment by ktbartholomew** [comment]
@cneill We really enjoy having these security-themed posts! I think it's important for developers who aren't typically focused on security to have some exposure to what's going on in that world.

I really like *everything* about what you're doing with these posts, especially how you provide commentary and context for each article you link to. That commentary adds a lot of value, and I don't think the "link dump" moniker does it justice. When I see that title my first impression is that the entire post is just a list of links and nothing more, but you're doing a lot more than that.

Would you consider naming this and all future posts something like "**This Week in Information Security**" or "**Weekly Security News Roundup**"? That more accurately captures the true value of what you're doing.

What do you think?

**Comment by cneill** [comment]
Thanks for the edits & title suggestions. The "Link Dump" name came from the days when I just used to email security stuff to my team roughly once a week, but I'm definitely a bigger fan of "This Week in Information Security."

I've changed the filename in my pull request, and made the suggested grammar edits. Let me know if you need anything else from me.

File: src/site_source/_posts/2015-05-25-this-week-in-information-security.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: "Weekly Security Link Dump (Week of May 25th)"
```
Comment (rgbkrk) [inline src/site_source/_posts/2015-05-25-this-week-in-information-security.md]: Jekyll front matter still needs updating.

**Comment by cneill** [comment]
Good catch! Updated to reflect the new name and publishing date. Let me know if I've missed anything else.

**Comment by rgbkrk** [comment]
:shipit:

**Comment by cneill** [comment]
Dangit, realized I didn't include a "more" tag this time. Would you like me to update it with one or just leave it be?

**Comment by rgbkrk** [comment]
You'll have to make a new PR and I'll merge it right away. My bad too.

**Comment by cneill** [comment]
Just submitted #1148

**Pull request merged**